File: C_Practice/bintodec.c
#include<stdio.h>
#include<string.h>
int main()
{
	int num = 10110110;
	int temp=0,rem=0,res=0,pow=1;
	int i;
	while(num!=0)
{
	rem = num%10;
	res = res+rem*pow;
	pow*=2;
	num = num/10;
}
printf("%d\n",res);
}

File: Java/bintodec.java
public class bin
{
	public:
		int num;
		int res,pow,rem;
		public void bintodec()
		{

			while(num!=0)
			{
				rem = num%10;
				res = res+rem*pow;
				pow*=2;
				num = num/10;
			}
			cout<<"num is "<<res<<endl;

		}
		public static void  main(Strins args[])
		{
			bin test;
			test.num=10110110;
			test.pow=1;
			test.bintodec();

		}
}

